Automattic/jetpack
# checks: Sync `engines` across all package.json files

#### Changes proposed in this Pull Request:

The `engines` key in package.json specifies acceptable versions of node
and yarn, and should be the same in all package.json files in the
monorepo. Add a check to ensure that's the case.

#### Jetpack product discussion


Came up in passing in review of #19705.

#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






* Is CI happy?
* Try changing or removing the `engines` value in a `package.json`, see that it reports errors properly.
